R/helpers.R

```diff
@@ -0,0 +1,25 @@
+#' Assert function package dependencies
+#'
+#' Asserts that `Suggests` packages are available for function to run.
+#'
+#' @param pkgs `[character()]` giving the required packages.
+#'
+#' @noRd
+assert_deps <- function(pkgs) {
```

Comment:
You could use `check_suggested()` in pkgload if you want https://github.com/r-lib/pkgload/blob/master/R/utils.R#L24

Reply:
yes!
Damn me, I was looking for something like that and wanted to ask you later 😄

Reply:
logged in https://github.com/r-lib/ghactions/issues/237